src/__tests__/components/Users.test.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import Users, {
+  Users as UsersComponent,
+  mapStateToProps,
+  mapDispatchToProps,
+} from '../../components/Users/Users';
+import usersMocks from '../../__mocks__/usersMocks';
+
+describe('Users', () => {
```

Comment:
Can you have `Users component` to make it more descriptive?

Reply:
Alright